src/Illuminate/Foundation/Console/Tinker/Presenters/EloquentCollectionPresenter.php

```diff
@@ -0,0 +1,35 @@
+<?php namespace Illuminate\Foundation\Console\Tinker\Presenters;
+
+use Psy\Presenter\ObjectPresenter;
+use Illuminate\Database\Eloquent\Collection;
+
+class EloquentCollectionPresenter extends ObjectPresenter {
+
+	/**
+	 * EloquentCollectionPresenter can present Collections.
+	 *
+	 * @param  mixed  $value
+	 * @return bool
+	 */
+	public function canPresent($value)
+	{
+		return $value instanceof Collection;
+	}
+
+	/**
+	 * Get an array of Collection object properties.
+	 *
+	 * \ReflectionProperty constants may be passed as $propertyFilter, and
```

Comment:
But they're constants on the ReflectionProperty class, and I don't feel like that would be as obvious without actually using the class name.